src/compiler/python_generator.cc

```diff
@@ -229,7 +229,8 @@ bool GetModuleAndMessagePath(const Descriptor* type,
   return true;
 }
 
-bool PrintServerFactory(const ServiceDescriptor* service, Printer* out) {
+bool PrintServerFactory(const std::string package_qualified_service_name,
```

Comment:
I believe the stylistically consistent signature is `const std::string&` rather than `const std::string` here; there's no particularly meaningful semantic difference in this case, but, y'know, consistency.